File: ATaxi.DL/Order.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ATaxi.DL
{
    [Table("Order")]
    public class Order
    {
        [Key]
        public int OrderId { get; set; }
        public DateTime? OrderDate { get; set; }
        public DateTime? OrderComplateDate { get; set; }
        [Required]
        public string OrderAddressSource { get; set; }
        [Required]
        public string OrderAddressDestination { get; set; }
        public int OrderStatus { get; set; }
        public int? CarID { get; set; }
    }
}

File: ATaxi/ClientApp/src/app/order.ts
import { Car } from './car';
export class Order {
    constructor(
        public orderId?: number,
        public orderDate?: Date,
        public orderComplateDate?: Date,
        public orderAddressSource?: string,
        public orderAddressDestination?: string,
        public orderStatus?: number,
        public carID?: number    ) { }
}